src/modules/pv/pv_core.c

```diff
@@ -3238,11 +3238,11 @@ int pv_get_msg_attrs(sip_msg_t *msg, pv_param_t *param, pv_value_t *res)
 
 int pv_parse_env_name(pv_spec_p sp, str *in)
 {
+	char *csname;
+
 	if(in->s==NULL || in->len<=0)
```

Comment:
I would do
```
if(sp==NULL || in == NULL || in->len <=0)
``` 

as in the [kamailio-devel-guide](http://www.asipto.com/pub/kamailio-devel-guide/#c16_add_pv)